Shopify/cli
# Fix `nginx.conf` conflict with `partners` in spin

### WHY are these changes introduced?

It looks like spin provides a default `nginx.conf` file when a repository does not provide its own. This default creates a server on `cli.<FQDN>`, but the `partners` `nginx.conf.erb` does the same. The `app dev` command relies on the `partners` version of `cli.<FQDN>` to be up and running, and the conflicting `nginx.conf` causes the spin provided default to take precedence over the `partners` one, causing the dev preview of the CLI to not work in spin.



### WHAT is this pull request doing?

This PR adds an empty `nginx.conf.erb` file to `.spin` to prevent spin from creating the default config which leads to the conflicting server entries.



### How to test your changes?

Make a new spin instance that with a 1p app and try to run `app dev`. Make sure the dev preview is accessible. You can use the `flow` spin constellation to try this out:

```
spin up flow:with-partners-and-extensions

# Then run this inside of the `flow` directory
yarn cli:dev
```



### Post-release steps



### Measuring impact

How do we know this change was effective? Please choose one:


### Checklist
